coresdk/src/test/unit_tests/unit_test_color.cpp

```diff
@@ -69,3 +69,70 @@ TEST_CASE("can convert a color to a hex string", "[color_to_string][color_green]
         REQUIRE(alpha_of(result) == 200);
     }
 }
+
+TEST_CASE("hsb_color converts HSB to RGB correctly", "[hsb_color]")
+{
+    SECTION("black when brightness is 0")
+    {
+        color black = hsb_color(0.5, 1.0, 0.0);
+        REQUIRE(red_of(black) == 0);
+        REQUIRE(green_of(black) == 0);
+        REQUIRE(blue_of(black) == 0);

```

Comment:
Check the alpha of the colour. HSB doesn't specify alpha so it should always be the max

```suggestion
        REQUIRE(red_of(black) == 0);
        REQUIRE(green_of(black) == 0);
        REQUIRE(blue_of(black) == 0);
        REQUIRE(alpha_of(black) == 255);
```